apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.html

```diff
@@ -1,36 +1,86 @@
-        </mat-icon>
-        <b>
-          {{member.status | transformMemberStatus}}
-        </b>
-      </td>
-    </tr>
-    <!-- TODO fix authorization of expiration attribute when it is supported-->
-    <tr *ngIf="authResolver.isThisVoAdminOrObserver(vo.id)">
-      <td class="vert-center">{{'MEMBER_DETAIL.OVERVIEW.EXPIRATION' | translate}}</td>
-      <td>
-        <i>
-          {{expiration}}
-        </i>
-        <button
-          *ngIf="authResolver.isThisVoAdmin(vo.id)"
-          (click)="changeExpiration()"
-          mat-icon-button>
-          <mat-icon>edit</mat-icon>
-        </button>
-      </td>
-    </tr>
-    </tbody>
-  </table>
 
-  <perun-web-apps-menu-buttons-field [items]="navItems" [size]="'small'"></perun-web-apps-menu-buttons-field>
+  <div>
+    <div class="align-cards">
+      <mat-card class="mat-elevation-z3">
+        <mat-card-content>
+          <table mat-table [dataSource]="dataSource">
+            <ng-container matColumnDef="attName">
+              <th *matHeaderCellDef mat-header-cell></th>
+              <td *matCellDef="let attribute" class="font-weight-bold" mat-cell>
+                {{attribute | attributeFriendlyName}}:
+              </td>
+            </ng-container>
+            <ng-container matColumnDef="attValue">
+              <th *matHeaderCellDef mat-header-cell></th>
+              <td *matCellDef="let attribute" class="column-center" mat-cell>
+                {{attributes.get(attribute)}}
+              </td>
+            </ng-container>
+            <tr
+              *matRowDef="let attribute; columns: displayedColumns;"
+              class="dark-hover-list-item"
+              mat-row
+            >
+            </tr>
+          </table>
+        </mat-card-content>
+      </mat-card>
+    </div>
+    <div class="align-cards">
+      <mat-card class="mat-elevation-z3">
+        <mat-card-header>
+          <mat-card-title>
+            <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.MEMBERSHIP' | translate}}</h1>
+          </mat-card-title>
+        </mat-card-header>
+        <mat-card-content>
+          <table mat-table [dataSource]="membershipDataSource">
+            <ng-container matColumnDef="attName">
+              <th *matHeaderCellDef mat-header-cell></th>
+              <td *matCellDef="let attribute" class="font-weight-bold" mat-cell>
+                {{attribute}}:
+              </td>
+            </ng-container>
+            <ng-container matColumnDef="attValue">
+              <th *matHeaderCellDef mat-header-cell></th>
+              <td *matCellDef="let attribute" class="column-center" mat-cell>
+                <div *ngIf="attribute === 'Status'">
+                  <mat-icon class="mr-1" [color]="statusIconColor">
+                    {{statusIcon}}
+                  </mat-icon>
+                  <b class="column-center">
+                    {{member.status | transformMemberStatus}}
+                  </b>
+                </div>
```

Comment:
The position of the icon and the text next to it looks weird
![image](https://user-images.githubusercontent.com/32871167/98552770-291f6580-229f-11eb-8ad9-2dd7d8a6fb58.png)
How I would like it to look
![image](https://user-images.githubusercontent.com/32871167/98552834-3b999f00-229f-11eb-9763-6a31d5ae8cdc.png)